pkg/monitoring/daemonsets_test.go

```diff
@@ -23,7 +23,7 @@ func TestCreateFilebeatDaemonSet(t *testing.T) {
 		constants.FilebeatName:     true,
```

Comment:
copyright 2021

Reply:
Addressed in commit e0cb7e9523fa6287222cc134a03a69a0cfd26fa2